### DISCUSSION TITLE
Tutorial for Nc Process Model not working

### DISCUSSION THREAD

**Discussion by Andy-Urbano** [lava-nc/lava#571] [Q&A] [answered]
Hello,
I am currently undertaking a project using Lava to program a neural network. I have access to the INRC servers and I'm trying to figure out how to run networks on the Loihi chip. The 3rd section of tuttorial00_tour_through_lava has code that is meant to exemplify this, but when I run it it gets stuck on the cell where the network is instantiated and run. When I check the SLURM job scheduler, it shows that a job is being run, but even when I reduce the number of steps to 10, it doesn't produce output or error messages for several minutes.

If anyone could tell me what I may be doing wrong and how to fix it, I would greatly appreciate it.
Thank you in advance for any help.

**Comment by weidel-p** [comment, marked as answer]
Hi, 

I'm sorry you're experiencing these issues. I just double checked that this tutorial runs through with the latest version of Lava and Lava-Loihi. May I ask you which versions of Lava and Lava-Loihi you are using? 

Also, you mentioned that the SLURM scheduler shows that a job is being executed. Could you paste that output here? Sometimes it happens that boards break or need to be rebooted. Could you try to specify to use another board manually to see if the issue persists among several boards?

**Reply by Andy-Urbano** [reply]
![Screenshot (27)](https://user-images.githubusercontent.com/119442780/212882781-0422fe9b-22c7-46ef-8381-e0a4d9618e72.png)

I'm using version 0.6.1 of Lava-Loihi. How do I specify the use of another board to try this?

**Reply by weidel-p** [reply]
Okay, great. 

There are multiple options to specify a board manually, the easiest is probably to set the LOIHI_GEN, PARTITION and BOARD environment variables when starting your jupyterlab.

You can run `sinfo` to see the available boards. Then you can pick one which is available and start your jupyerlab like this:

`SLURM=1 LOIHI_GEN=N3B3 PARTITION=oheogulch BOARD=ncl-ext-og-01 jupyter-lab --port XXXX`

**Reply by Andy-Urbano** [reply]
It's working as intended now, thank you very much!
